src/content/docs/en/recipes/i18n.mdx

```diff
@@ -505,3 +505,4 @@ Translate the routes of your pages for each language.
 - [astro-i18next](https://github.com/yassinedoghri/astro-i18next) — An Astro integration for i18next including some utility components.
 - [astro-i18n](https://github.com/alexandre-fernandez/astro-i18n) — A TypeScript-first internationalization library for Astro.
 - [astro-i18n-aut](https://github.com/jlarmstrongiv/astro-i18n-aut) — An Astro integration for i18n that supports the `defaultLocale` without page generation. The integration is adapter agnostic and UI framework agnostic.
+- [paraglide](https://inlang.com/c/astro) - A fully typesafe i18n library that plays really well with islands.
```

Comment:
```suggestion
- [paraglide](https://inlang.com/c/astro) - A fully type-safe i18n library that works with islands.
```
I checked, and it seems that "type-safe" is the correct adjective form (I might need to check the rest of our site, too!) and we really need a more literal, neutral phrase that will be easier on our translators than "plays well with." Maybe "works with"? Do you have a better idea here?

Reply:
Good note on the "type-safe", thanks!

"plays well with" and "works with" don't convey the same meaning. "plays well with" implies a level of synergy, whereas "works with" only states that it does function. 

What do you think of the phrasing "A fully type-safe i18n library that's great for islands"?

Reply:
if it were "designed for islands", that's a feature worth stating.

Is there a particular way in which it's great for islands? A particular noteworthy feature to call out? (Honestly, my initial reaction was to delete that entirely. Unless you can tell me something significant about it that means we *should* mention islands, it's not really worth calling out.)

Reply:
Paraglide wasn't designed for Astros Islands in particular, but for any partial hydration pattern. 

With paraglide messages are treeshakeable. That way any island only adds exactly the messages that it needs to the JS bundle no more. Usually this would require developers to sort their messages into namespaces, but with Paraglide it's automatic.

Reply:
That's fits really well, great suggestion!
I'll change it to that. Thank you